src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -452,23 +465,38 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
 
         List<EmailWrapper> emails = new ArrayList<EmailWrapper>();
 
+        /* The instructor preamble aims at informing the instructor about an email that has been sent to the students
+        of his course */
```

Comment:
Should be a proper javadoc comment.
```java
/**
 * The preamble added to email copies sent to instructors
 */
```
In any case, I don't think we need this comment anymore because the variable name is explanatory enough.